docs/security/access-control.md

```diff
@@ -0,0 +1,663 @@
+| UseCatalog    | Metalake, Catalog         |                  |
+
+:::info
+
+`USE_CATALOG` is needed for a user to interact with any object within the catalog. 
+
+For example, to select data from a table, users need to have the SELECT_TABLE privilege on that table and
+
+`USE CATALOG` privileges on its parent catalog as well as `USE SCHEMA` privileges on its parent schema.
+
+:::
+
+
+### Schema privileges
+
+| Name         | Supports Securable Object | Operation       |
+|--------------|---------------------------|-----------------|
+| CreateSchema | Metalake, Catalog         | Create a schema |
+| UseSchema    | Metalake, Catalog, Schema | Use a schema    |
+
+:::info
+
+`UseSchema`is needed for a user to interact with any object within the schema. 
+
+For example, to select data from a table, users need to have the `SELECT_TABLE` privilege on that table
+
+and `USE SCHEMA` privileges on its parent schema.
+
+:::
+
+### Table privileges
+
+| Name        | Supports Securable Object         | Operation                                      |
+|-------------|-----------------------------------|------------------------------------------------|
+| CreateTable | Metalake, Catalog, Schema         | Create a table                                 |
+| ModifyTable | Metalake, Catalog, Schema, Table  | Use the SQL `UPDATE`,`DELETE`,`INSERT` a table |
+| SelectTable | Metalake, Catalog, Schema, Table  | Use the SQL `SELECT` data from a table         |
+
+### Topic privileges
+
+| Name         | Supports Securable Object        | Operation                                 |
+|--------------|----------------------------------|-------------------------------------------|
+| CreateTopic  | Metalake, Catalog, Schema        | Create a topic                            |
+| ProduceTopic | Metalake, Catalog, Schema, Topic | Produce a topic (including alter a topic) |
+| ConsumeTopic | Metalake, Catalog, Schema, Topic | Consume a topic                           |
+
+### Fileset privileges
+
+| Name          | Supports Securable Object          | Operation                                   |
+|---------------|------------------------------------|---------------------------------------------|
+| CreateFileset | Metalake, Catalog, Schema          | Create a fileset                            |
+| WriteFileset  | Metalake, Catalog, Schema, Fileset | Write a fileset (including alter a fileset) |
+| ReadFileset   | Metalake, Catalog, Schema, Fileset | read a fileset                              |
+
+## Inheritance Model
+
+Securable objects in Gravitino are hierarchical and privileges are inherited downward.
+
+This means that granting a privilege on a metalake, catalog or schema automatically grants
+
+the privilege to all current and future objects within the metalake, catalog or schema.
+
+For example, if you give a use that `SELECT_TABLE` privilege on a catalog, then that the user
+
+will be able to select(read) all tables in that catalog.
```

Comment:
This one-line sentence will break into two lines according to your current way of writing markdown.

Reply:
OK, I checked other places, too. Fix them by the way.